registry/src/skills.json

```diff
@@ -460,5 +460,55 @@
             "Synthesizing results",
             "Preparing insights"
         ]
+    }, 
+    {
+      "pluginId": "79449cd9-0590-4ccc-9d9f-12f99438c17b",
+      "name": "@moxie-protocol/plugin-degenfans-alfafrens",
+      "displayName": "DegenFans AlfaFrens",
+      "version": "0.0.1",
+      "author": "DegenFans",
+      "description": "Enables your agent to recommend staking options for AlfaFrens channels",
+      "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-degenfans-alfafrens",
+      "logoUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-degenfans-alfafrens/images/logo.png",
+      "settings": {},
+      "capabilities": ["show the best staking options for a given staking amount"],
+      "starterQuestions": [
+        {
+          "label": "What is the Degenfans Alfafrens Skills?",
+          "value": "What is the Degenfans Alfafrens Skills?"
+        },
+        {
+          "label": "What are some staking recommendations you have for Alfafrens?",
+          "value": "What are some staking recommendations you have for Alfafrens?"
+        },
+        {
+          "label": "I have 9000 AF — how should I stake it?",
+          "value": "I have 9000 AF — how should I stake it?"
+        },
+        {
+          "label": "What are your top picks for staking on Alfafrens?",
+          "value": "What are your top picks for staking on Alfafrens?"
+        }
+      ],
+      "mediaUrls": [
+        ""
+      ],
+      "actions":  [
+         "GET_ALFAFRENS_STAKING_RECOMENDATION",
+          "VIEW_ALFAFRENS_STAKING_RECOMENDATION",
+          "GIVE_ALFAFRENS_STAKING_RECOMENDATION",
+          "GET_ALFAFRENS_STAKING_CONSULTANT",
+          "STAKING_RECOMMENDATION_ON_ALFAFRENS",
+          "DEGENFANS_ALFAFRENS_INFO",
+          "WHAT_IS_DEGENFANS_ALFAFRENS",
+          "HOW_DOES_DEGENFANS_ALFAFRENS_WORK",
+          "HOW_TO_USE_DEGENFANS_ALFAFRENS"
+      ],
+      "isPremium": true,
+      "freeQueries": 0,
+      "skillCoinAddress": "0x112cf1CC540eaDF234158c0e4044C3b5F2A33e5e",
+      "minimumSkillBalance": 5,
+      "status": "ACTIVE",
+      "isDefault": false
```

Comment:
Other than that, there's also new fields `loaders` which you can customize the loader text when loading the response as shown in the screen here:

<img width="551" alt="Screenshot 2025-04-02 at 22 29 43" src="https://github.com/user-attachments/assets/8a1a2416-240a-4551-8888-56a8e4cf37ed" />

Simply just add like this:
```suggestion
      "isDefault": false,
      "loaders": [
            "Text 1",
            "Text 2"
        ]
    },
```

This is optional, so if you don't want to, feel free to skip it